pkgs/by-name/fi/fireplace/package.nix

```diff
@@ -0,0 +1,39 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  ncurses,
+}:
+stdenv.mkDerivation {
+  pname = "fireplace";
+  version = "0-unstable-2020-02-02";
+
+  buildInputs = [ ncurses ];
+
+  installPhase = ''
+    runHook preInstall
+
+    install -Dm555 fireplace -t $out/bin
+
+    runHook postInstall
+  '';
+
+  src = fetchFromGitHub {
+    owner = "Wyatt915";
+    repo = "fireplace";
+    rev = "aa2070b73be9fb177007fc967b066d88a37e3408";
+    hash = "sha256-2NUE/zaFoGwkZxgvVCYXxToiL23aVUFwFNlQzEq9GEc=";
+  };
+
+  meta = {
+    description = "Cozy fireplace in your terminal";
+    homepage = "https://github.com/Wyatt915/fireplace";
+    license = lib.licenses.mit;
+    maintainers = with lib.maintainers; [
+      multivac61
+      eclairevoyant
```

Comment:
This is a low-priority leaf package written in C, so I'm fine to maintain this, but next time, if you're adding another person's name, please ask if they're willing to maintain first :slightly_smiling_face:

Reply:
Sorry about that, [I misunderstood the above comment](https://github.com/NixOS/nixpkgs/pull/340389#discussion_r1749016514) and your 👍 emoji reaction.

Reply:
🤗